docs/applications/deploy-application-using-incoming-hook.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Deploy Application using an Incoming Hook
+intro: As project users base grow, there's a need to rely more on processes and automation; Devopness helps by providing, among other tools, customizable CI/CD pipelines, empowering our users to automate common and repetitive tasks to improve code quality and their code review processes. Create a incoming webhook to trigger a application deploy programmatically.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+Before creating our application deploy incoming webhook, we need the `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`) to use as parameters in our API calls.
+
+<details open>
+  <summary>Steps to find `Application ID` and `Deploy Pipeline ID`</summary>
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. In the list of applications, find the application with the pipeline you want to manage and click `DETAILS`
+1. On the upper-right corner of the applications details view click `SETTINGS`
+1. Use the drop-down menu to choose `PIPELINES`
+1. In the list of pipelines, find the pipeline you want to manage and click `DETAILS`
+    > If you haven't created a pipeline yet, follow the [Add a Pipeline](/docs/pipelines/add-pipeline) guide
+1. Copy the `Application ID` and `Deploy Pipeline ID` from the URL, considering the following URL format:
+    ```bash
+    https://app.devopness.com/projects/<project_id>/environments/<environment_id>/applications/<application_id>/pipelines/<pipeline_id>
+    ```
+
+</details>
+
+Once you have your `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`), please follow the instructions below to add an incoming webhook to your application that triggers the deploy pipeline:
+
+1. Follow the steps on [Create an Incoming Webhook](/docs/webhooks/create-incoming-webhook) guide 
+1. Once your web hook is created, copy the webhook the hook's unique URL (`url` field)
+1. Take note of the hook secret signature key (`secret` field)
+  > If the hook was created without a `secret` (non recommended), you can skip this step
+1. Add your webhook to your source provider
+    - Follow Bitbucket' [Manage webhooks: create webhooks](https://support.atlassian.com/bitbucket-cloud/docs/manage-webhooks/#Create-webhooks) guide; or
```

Comment:
I believe the links should be easier to maintain and the usage of `guide, or` in the end of each item of the bullet list will make the lines harder to edit.

Suggestions below:

~~1. Add your webhook to your source provider~~
1. Add your webhook to the source provider where each application's source code is hosted, by following the source provider's instructions on the links below:

```Suggestion
- Bitbucket: [Manage webhooks: create webhooks](https://support.atlassian.com/bitbucket-cloud/docs/manage-webhooks/#Create-webhooks)

- Github: [Webhooks: setting up a webhook](https://docs.github.com/en/webhooks-and-events/webhooks/creating-webhooks#setting-up-a-webhook)

- Gitlab: [Webhooks: configure a webhook in GitLab](https://docs.gitlab.com/ee/user/project/integrations/webhooks.html#configure-a-webhook-in-gitlab)
```

Reply:
Suggestion Applied